index.d.ts

```diff
@@ -0,0 +1,293 @@
+/// <reference types="request" />
+/// <reference types="node" />
+
+import * as request from 'request';
```

Comment:
Not going to argue this one, but it extends `request-promise` which I don't think has types anyway